### PR TITLE
testing: enable Crucible in FPR

### PR DESCRIPTION
**Pull request by jasuarez**
Add support for running crucible testsuite in full-piglit-run.sh script

### CODE REVIEW

File: testing/full-piglit-run.sh [hunk 1]
```diff
@@ -517,17 +551,20 @@ Options:
   --piglit-results-path <path>     <path> to the piglit results
   --vk-gl-cts-path <path>          <path> to the built vk-gl-cts binaries
   --deqp-path <path>               <path> to the built dEQP binaries
+  --crucible-path <path>           <path> to the built crucible binaries
   --vk-cts-prefix <prefix>         <prefix> to use with the vk-cts run
   --gl-cts-prefix <prefix>         <prefix> to use with the gl-cts run
   --deqp-gles2-prefix <prefix>     <prefix> to use with the dEQP GLES2 run
   --deqp-gles3-prefix <prefix>     <prefix> to use with the dEQP GLES3 run
   --deqp-gles31-prefix <prefix>    <prefix> to use with the dEQP GLES3.1 run
+  --crucible-prefix <prefix>       <prefix> to use with the dEQP GLES3.1 run
```
Comment (tanty) [inline testing/full-piglit-run.sh]: The usage documentation seems wrong.

**Review by tanty — CHANGES_REQUESTED** [review]
Other than the comment inline, LGTM.

**Comment by jasuarez** [comment]
Changes done

File: testing/full-piglit-run.sh [hunk 2]
```diff
@@ -517,17 +551,20 @@ Options:
   --piglit-results-path <path>     <path> to the piglit results
   --vk-gl-cts-path <path>          <path> to the built vk-gl-cts binaries
   --deqp-path <path>               <path> to the built dEQP binaries
+  --crucible-path <path>           <path> to the built crucible binaries
   --vk-cts-prefix <prefix>         <prefix> to use with the vk-cts run
   --gl-cts-prefix <prefix>         <prefix> to use with the gl-cts run
   --deqp-gles2-prefix <prefix>     <prefix> to use with the dEQP GLES2 run
   --deqp-gles3-prefix <prefix>     <prefix> to use with the dEQP GLES3 run
-  --deqp-gles31-prefix <prefix>    <prefix> to use with the dEQP GLES3.1 run
+  --deqp-gles31-prefix <prefix>    <prefix> to use with the crucible run
```
Comment (tanty) [inline testing/full-piglit-run.sh]: You have swapped the documentation incorrectly. Please fix this.

Additionally, you can just force pull the commit and have 1 single commit, and not 2 (or 3).

**Review by tanty — CHANGES_REQUESTED** [review]

**Comment by jasuarez** [comment]
Pushed again with the fix.

**Review by tanty — APPROVED** [review]

**Comment by tanty** [comment]
LGTM

:+1:

**Pull request merged**